services/api/src/models/definitions/user.json

```diff
@@ -33,8 +33,11 @@
       "required": true,
       "trim": true
     },
-    "timeZone": {
-      "type": "String"
+    "phoneNumber": {
+      "type": "String",
+      "required": true,
+      "validate": "phoneNumber",
+      "trim": true
```

Comment:
A unique field would be nice here ... but might have to wait for it

Reply:
added a ticket for that and will have a think about it, we should be able to do it